x-pack/plugins/security_solution/public/overview/components/detection_response/host_alerts_table/host_alerts_table.tsx

```diff
@@ -0,0 +1,172 @@
+
+import React, { useMemo } from 'react';
+
+import {
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiHealth,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { SecurityPageName } from '../../../../app/types';
+import { HeaderSection } from '../../../../common/components/header_section';
+import { useQueryToggle } from '../../../../common/containers/query_toggle';
+import { useNavigation, NavigateTo, GetAppUrl } from '../../../../common/lib/kibana';
+import * as i18n from '../translations';
+import { SEVERITY_COLOR } from '../util';
+import { AlertSeverityCounts, useHostAlertsItems } from './host_alerts_items';
+
+type GetTableColumns = (params: {
+  getAppUrl: GetAppUrl;
+  navigateTo: NavigateTo;
+}) => Array<EuiBasicTableColumn<AlertSeverityCounts>>;
+
+const DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID = 'vulnerableHostsBySeverityQuery';
+
+export const HostAlertsTable = React.memo(() => {
+  const { getAppUrl, navigateTo } = useNavigation();
+  const { toggleStatus, setToggleStatus } = useQueryToggle(
+    DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID
+  );
+
+  const { data, isLoading } = useHostAlertsItems({
+    skip: !toggleStatus,
+    queryId: DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID,
+  });
+
+  const columns = useMemo(
+    () => getTableColumns({ getAppUrl, navigateTo }),
+    [getAppUrl, navigateTo]
+  );
+
+  return (
+    <EuiPanel hasBorder data-test-subj="hostSeverityAlertsPanel">
+      <HeaderSection
+        id={DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID}
+        title={i18n.HOST_ALERTS_SECTION_TITLE}
+        titleSize="s"
+        hideSubtitle
+        toggleStatus={toggleStatus}
+        toggleQuery={setToggleStatus}
+      />
+      {toggleStatus &&
+        (isLoading || data.length > 0 ? (
+          <>
+            <EuiBasicTable
+              data-test-subj="hostAlertsTable"
+              columns={columns}
+              items={data}
+              loading={isLoading}
+            />
+            <EuiSpacer size="m" />
+            <EuiButton
+              onClick={() => navigateTo({ url: getAppUrl({ deepLinkId: SecurityPageName.hosts }) })}
+            >
+              {i18n.VIEW_ALL_HOST_ALERTS}
+            </EuiButton>
+          </>
+        ) : (
+          <EuiEmptyPrompt title={<h3>{i18n.NO_ALERTS_FOUND}</h3>} titleSize="xs" />
+        ))}
+    </EuiPanel>
+  );
+});
+
+HostAlertsTable.displayName = 'HostAlertsTable';
+
+const getTableColumns: GetTableColumns = ({ getAppUrl, navigateTo }) => [
+  {
+    field: 'hostName',
+    name: i18n.HOST_ALERTS_HOSTNAME_COLUMN,
+    truncateText: true,
+    textOnly: true,
+    render: (hostName: string) => {
+      const url = getAppUrl({ deepLinkId: SecurityPageName.hosts, path: hostName });
+
+      return (
+        <EuiLink
+          data-test-subj="userSeverityAlertsTable-hostName"
+          href={`/app/security/hosts/${hostName}`}
```

Comment:
```suggestion
          href={url}
```

Reply:
I found we have an existing ```<HostDetailsLink hostName={... />```
consumed by `x-pack/plugins/security_solution/public/hosts/components/hosts_table/columns.tsx` line 60.
Could we reuse that?

Reply:
I did use it, nice find!, using for the users one as well